knowlg-automation/terraform/aws/variables.tf

```diff
@@ -0,0 +1,228 @@
+variable "env" {
+  type        = string
+  description = "Environment name. All resources will be prefixed with this value."
+  default     = "dev"
+}
+
+variable "building_block" {
+  type        = string
+  description = "Building block name. All resources will be prefixed with this value."
+  default     = "knowlg"
+}
+variable "additional_tags" {
+  type        = map(string)
+  description = "Additional tags for the resources. These tags will be applied to all the resources."
+  default     = {}
+}
+variable "region" {
+  description = "AWS region for resources"
+  default     = "us-west-1" # Replace with your desired AWS region
+}
+variable "cluster_name" {
+  description = "Name of the EKS cluster"
+  default     = "my-eks-cluster" # Replace with your desired cluster name
```

Comment:
Please give proper cluster name

Reply:
Eks-cluster would be fine to do so in the var file i think?

Reply:
Take the reference from azure provision cluster name.